src/modules/mealDisplay.js

```diff
@@ -0,0 +1,103 @@
+import popupComment from './display-popup-comment';
+import count from './items-count';
+import newLike from './new-like';
+
+async function fetchLikesData(apikey, likeid) {
```

Comment:
As per the project requirements, you should be using ES6 features, that include the use of arrow functions instead of the old function declaration way. Kindly update your code accordingly.